src/components/Table/GridTable.tsx

```diff
@@ -80,6 +82,8 @@ export interface GridStyle {
   emptyCell?: ReactNode;
   presentationSettings?: Pick<PresentationFieldProps, "borderless" | "typeScale"> &
     Pick<PresentationContextProps, "wrap">;
+  /** Minimum table width in pixels. Used when calculating columns sizes */
+  minWidth?: number;
```

Comment:
I thought about either having this defined directly here, or grabbing the minWidth from `GridStyle.rootCss.minWidth`. I decided to go this route in order to define that the property must be a `number`/pixels. Otherwise we'd need to do more checks to ensure that we only accept pixel or percentage values. Though, I don't think a use case for use a percentage minWidth exists, as the default percentage minWidth would be 100% and that seems sufficient. This value will always override any value defined in `rootCss`.